src/kirby/components/list/list-item-color.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input, Renderer2, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
+
+@Directive({
+  selector: '[kirbyListItemColor]',
+})
+export class ListItemColorDirective implements OnChanges {
+  @Input()
+  colorCb: (item: any) => string;
```

Comment:
Consider aliasing this with the directive name, so it can be applied like:
`kirbyListItemColor="myColorCallback"`

Reply:
Done.